vms/avm/vm.go

```diff
@@ -591,39 +573,6 @@ func (vm *VM) lookupAssetID(asset string) (ids.ID, error) {
 // Invariant: any error returned by onAccept should be considered fatal.
 // TODO: Remove [onAccept] once the deprecated APIs this powers are removed.
 func (vm *VM) onAccept(tx *txs.Tx) error {
```

Comment:
Should this function no longer return an `error`?

Reply:
Removed https://github.com/ava-labs/avalanchego/pull/3872/commits/c6124b30f3258eebe7f2eeb57c3c488550f3791d